internal/broker/server.go

```diff
@@ -124,8 +125,8 @@ func (srv *Server) run(ctx context.Context, addr string, listenAndServe func(srv
 func (srv *Server) CreateHandler() http.Handler {
 	var rtr = mux.NewRouter()
 
-	rtr.HandleFunc(health.HandleBrokerLive()).Methods("GET")
-	rtr.HandleFunc(health.HandleReady()).Methods("GET")
+	rtr.HandleFunc(health.BrokerReadyProbe(srv.etcdURL)).Methods("GET")
```

Comment:
I wonder is not better to create and register probes outside the broker server instead of push all necessary arguments into the broker to just register probes. For now, maybe it is not that urgent but if we will have more probes then inject all necessary arguments is a bad idea.